client/resource_manager_client.go

```diff
@@ -119,7 +99,11 @@ func (c *client) ListResourceGroups(ctx context.Context, ops ...GetResourceGroup
 	}
 	req := &rmpb.ListResourceGroupsRequest{
 		WithRuStats: getOp.withRUStats,
-		KeyspaceId:  getOp.withKeyspaceID,
+	}
+	if c.inner.keyspaceID != constants.NullKeyspaceID {
```

Comment:
It seems unnecessary to check whether it is `NullKeyspaceID`; as a new implementation, the current configured keyspace ID should be passed in any case.